x11-plugins/compiz-plugins-main/compiz-plugins-main-0.8.14.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="Compiz Window Manager: Plugins"
+HOMEPAGE="https://github.com/compiz-reloaded"
+SRC_URI="https://github.com/compiz-reloaded/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	gnome-base/librsvg
+	virtual/jpeg:0
+	virtual/glu
+	>=x11-libs/compiz-bcop-0.7.3
+	<x11-libs/compiz-bcop-0.9
+	>=x11-wm/compiz-0.8
+	<x11-wm/compiz-0.9
+	x11-libs/cairo
```

Comment:
According to the configure scripts, `cairo-plugins-main` and `cairo-plugins-experimental` require `x11-libs/cairo[X]`.